linux_os/guide/system/permissions/files/dir_perms_world_writable_root_owned/bash/shared.sh

```diff
@@ -1,3 +1,5 @@
 # platform = multi_platform_all
 
-find / -not -fstype afs -not -fstype ceph -not -fstype cifs -not -fstype smb3 -not -fstype smbfs -not -fstype sshfs -not -fstype ncpfs -not -fstype ncp -not -fstype nfs -not -fstype nfs4 -not -fstype gfs -not -fstype gfs2 -not -fstype glusterfs -not -fstype gpfs -not -fstype pvfs2 -not -fstype ocfs2 -not -fstype lustre -not -fstype davfs -not -fstype fuse.sshfs -type d -perm -0002 -uid +0 -exec chown root {} \;
+# At least under containerized env /proc can have files w/o possilibity to
```

Comment:
The content of this commit is also part of https://github.com/ComplianceAsCode/content/pull/10563. Please, remove it here.

Reply:
I removed extra commits